leankyr/Warmup_Problems/Utopian_Tree/ReadMe.txt

```diff
@@ -0,0 +1,4 @@
+Tried to Use Reduction and One Liner to Replace fun UtopianTree
+did not work
+I think it was because both Reduction and One liner do not take into
+consideration Base Case n==0
```

Comment:
You are right and this is why I initialize the reducer with 1 both [here](https://github.com/primebotio/python-training-camp/pull/9/files#diff-e8b3c65d0df60a69c762743d149ecbb7R67) and [here](https://github.com/primebotio/python-training-camp/pull/9/files#diff-e8b3c65d0df60a69c762743d149ecbb7R64) it doesn't work?